andela/king-kong-ah-backend
# #166272091 setup sequelize

#### What does this PR do?
This PR sets up sequelize for author's haven 

#### Description of Task to be completed?
- install sequelize and sequelize-cli
- initialize sequelize
- setup config.js with environment variables
- remove files associated with moongoose
- add *.sqlite gitignore
- setup sqlite as test database

#### How should this be manually tested?

 - run `$ git clone https://github.com/andela/king-kong-ah-backend.git`
 - switch to the `ch-setup-sequelize-166272091` branch
 - run `$ npm install`
 - run `$ npm run start-dev`
 -  the message `connection to DB has been successfully established`  should be logged to the console
  
  
#### background context

- when `$ npm test` is run, a database.sqlite is created at the root directory if it does not already exist



#### What are the relevant pivotal tracker stories?

- PT Story link - [#166272091](https://www.pivotaltracker.com/story/show/166272091)



#### Screenshots
- Screenshot showing a successful connection when "$ npm run start-dev" is run 
<img width="363" alt="Screenshot 2019-06-01 at 5 16 38 PM" src="https://user-images.githubusercontent.com/40043468/58752845-a68d5680-84ad-11e9-9091-f010c4c86559.png">
